notebooks/04-Conditionals.ipynb

```diff
@@ -1216,13 +1205,14 @@
    ]
   },
   {
-   "attachments": {},
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {
     "slideshow": {
      "slide_type": "slide"
     }
    },
+   "outputs": [],
    "source": [
     "There's a lot of flexibility!"
    ]
```

Comment:
Looks like the cell got change to markdown, although I feel like that string would be fine as plain Python?